lib/nylas/api.rb

```diff
@@ -36,6 +36,22 @@ def authenticate(name:, email_address:, provider:, settings:, reauth_account_id:
       )
     end
 
+    # @return [String] A url for hosted authentication
+    def hosted_authentication_url(redirect_uri:, login_hint: "", response_type: "code",
+                                  scope: "email", state: nil)
+      params = {
+        client_id: app_id,
+        redirect_uri: redirect_uri,
+        response_type: response_type,
+        scope: scope,
```

Comment:
should this be `scopes:` instead?

Reply:
I think it should be `scopes:`. Updated!